incubator/mongodb/templates/deployment.yaml

```diff
@@ -56,8 +52,13 @@ spec:
           initialDelaySeconds: 5
           timeoutSeconds: 1
         volumeMounts:
-        - name: mongodb-data
+        - name: data
           mountPath: /bitnami/mongodb
       volumes:
```

Comment:
Can you add a resources block like in https://github.com/kubernetes/charts/blob/master/stable/redis/templates/deployment.yaml#L43?